src/hooks/useTransactionViolations.ts

```diff
@@ -5,7 +5,7 @@ import type {TransactionViolation, TransactionViolations} from '@src/types/onyx'
 import getEmptyArray from '@src/types/utils/getEmptyArray';
 import useOnyx from './useOnyx';
 
-function useTransactionViolations(transactionID?: string): TransactionViolations {
+function useTransactionViolations(transactionID?: string, shouldShowRterForSettledReport = true): TransactionViolations {
```

Comment:
```suggestion
function useTransactionViolations(transactionID?: string): TransactionViolations {
```

Let's not use param for it as I believe it should apply everywhere unless you find any specific scenario to show it.